rules-reviewed/eap7/eap6/hibernate4.windup.xml

```diff
@@ -604,5 +572,160 @@
                 <matches pattern="(TableGenerator|TableHiLoGenerator|SequenceGenerator|SequenceIdentityGenerator|SequenceHiLoGenerator)" />
             </where>
         </rule>
+        <rule id="hibernate4-00029">
+            <when>
+                <javaclass references="org.hibernate.type.descriptor.sql.SqlTypeDescriptor">
```

Comment:
Yes, it needs them on the library path in some way in order for it to match transitively.